### ISSUE TITLE
Robot Arms Pub Visual Identity

### ISSUE THREAD

**Issue by wayspurrchen** [RobotArmsClub/RobotArmsClub#6]
Related to #3. I personally envision the RAP visual identity as being cyberpunky, and technology-driven without being too on the nose. In my mind it should look as if a corporation went evil and knows it; fairly sleek with a bit of edge. I'm open to other concepts / descriptions to capture hearts and minds, but here are some links and resources that I like so far.
- Brutalist websites - http://brutalistwebsites.com/. I really like some of these. Some of them are a bit too alien and not easy to engage with; I want the mood to feel seductive and a alien, but not alienating.
  - http://developer.apple.com/wwdc
  - http://danieldittmar.net/
- Much of [Archillect](http://archillect.com/) is incredibly dope but perhaps too austere.

For some reason, really bold colors of pink, green, and black are coming to me.
# Image Dump

![](https://40.media.tumblr.com/e28025fac2ec5738a871a6c9d2f870f9/tumblr_o67pxnqArs1v8g6p5o1_540.jpg)
![](https://40.media.tumblr.com/053cd2e2ffce00f1f4340e2ccd35e83d/tumblr_o67nsr2U7M1re74mto1_540.jpg)
![](https://40.media.tumblr.com/72176ac54bd0893868e6cc3c8168a811/tumblr_o67lukzCCA1vrvt9fo1_540.jpg)
![](https://41.media.tumblr.com/efbe436c3e0e1f2a070a6724923711af/tumblr_o643f44kZ71tm1yjlo2_540.jpg)
![](https://s-media-cache-ak0.pinimg.com/736x/e0/16/2c/e0162c8d38eb10d25f703685049a0f0f.jpg)

**Comment by flipactual** [comment]
![](https://images.bigcartel.com/bigcartel/product_images/161521987/max_h-1000+max_w-1000/warmachinescan.jpg)
![](http://static1.squarespace.com/static/5006009984ae2a1f6383c99b/54604a34e4b06e0eeca4cf93/556fa4efe4b0467afc6a5277/1460595701191/fury.jpg?format=1000w)
![brettparsonstgbooga](https://cloud.githubusercontent.com/assets/1306968/14802217/3edfac64-0b15-11e6-9f8d-de4e39a21592.jpg)

**Comment by tek-witch** [comment]
I can produce cover visuals indefinitely for basically no money (which is going to be relevant, because there's a lot of awesome out there, but not a lot of free awesome), but my visual style is peculiar, and may not be awesome enough.

  If it doesn't work for the group, we'll need think about who/how we intend to license art from, keeping legal ducks in a row, record keeping, all that.

A little bit of my stuff:

http://gbchew.deviantart.com/gallery/?catpath=/

**Comment by wayspurrchen** [comment]
@exprat Cool stuff! I'm sure we can use it here and there - I'm also thinking of dusting off my pencils for this... I'll need to get my hands on a scanner.

We can also do some community outreach for submissions once we've got the website up and looking real.

**Comment by flipactual** [comment]
I vote for a brutalist cover with retro future colors. But I like minimal things.

**Comment by beaugunderson** [comment]
![image](https://cloud.githubusercontent.com/assets/61791/14804456/74155c64-0b31-11e6-817d-c0de79086ff9.png)

**Comment by beaugunderson** [comment]
also [this artist](http://doeggy.tumblr.com/tagged/art%20tag) is one of my favs and i think is available for work (some nsfw images there)

**Comment by flipactual** [comment]
tbh I'm inclined to set a deadline for discussion about visual artistic direction at which point we appoint and defer to the judgment of someone placed in charge of that area. Otherwise I fear an infinite bikeshed awaits.

**Comment by r4d1n** [comment]
I agree, but I think we have some time to share visual fodder and see what grows out of it.

**Comment by wayspurrchen** [comment]
+1. Let's not take this too seriously and just have fun with it before we get too serious. :)
